src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
        }
```

create constant NUMBER_OF_BALLS

Reply:
I added constant, but after 'mvn clean package' command, i got error .java:6:19: Abbreviation in name 'NUMBER_OF_BALLS' must contain no more than '2' consecutive capital letters. [AbbreviationAsWordInName]. I tried using NUM_BALLS but i got same error.

Reply:
<img width="1324" alt="Screenshot 2024-06-18 at 08 44 32" src="https://github.com/mate-academy/jv-lottery/assets/14053156/6bd91ec0-c9e1-4520-90c7-5cfa281e9f9a">

I haven't got any errors